docxcompose/properties.py

```diff
@@ -6,7 +6,7 @@
 from docx.opc.oxml import serialize_part_xml
 from docx.opc.packuri import PackURI
 from docx.opc.part import Part
-from docx.oxml import parse_xml
+from docx.oxml.parser import parse_xml
```

Comment:
I installed latest version of docx, and line 11 of docx/oxml/__init__.py is
```py
from docx.oxml.parser import OxmlElement, parse_xml, register_element_cls
```
That is to say `from docx.oxml.parser import parse_xml` is equal to `from docx.oxml import parse_xml`